bundles/org.openhab.binding.androiddebugbridge/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,403 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="androiddebugbridge"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="android">
+		<label>Android Device Thing</label>
+		<description>Android Device Thing for Android Debug Bridge Binding</description>
+		<channels>
+			<channel id="key-event" typeId="key-event-channel"/>
+			<channel id="text" typeId="text-channel"/>
+			<channel id="media-volume" typeId="media-volume-channel"/>
+			<channel id="media-control" typeId="media-control-channel"/>
+			<channel id="start-package" typeId="start-package-channel"/>
+			<channel id="stop-package" typeId="stop-package-channel"/>
```

Comment:
That channel allows to close an application by package name either if it is running in the background or not, I would like to keep like this. I'm using it in a rule right now to close the previous application when I send a new one, there I can add extra logic to avoid close the launcher or certain applications.
I will add another channel stop-current-package that works as you mention because you are right it could be more useful and simple in many situations.